apps/financial-aid/web-veita/src/components/MunicipalityAdminSettings/MunicipalityNumberInput/MunicipalityNumberInput.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box } from '@island.is/island-ui/core'
+import { NumberInput } from '@island.is/financial-aid-web/veita/src/components'
+import { AidName, KeyMapping } from '@island.is/financial-aid/shared/lib'
+
+interface Props {
+  id: string
+  aid: number
+  prefix: string
+  error: boolean
+  update: (value: number) => void
+}
+
+const MunicipalityAdminInput = ({ id, aid, prefix, error, update }: Props) => {
+  const label: KeyMapping<AidName, string> = {
+    ownPlace: 'Eigið húsnæði',
+    registeredRenting: 'Leiga með þinglýstan leigusamning',
+    unregisteredRenting: 'Leiga með óþinglýstan leigusamning',
+    withOthers: 'Býr eða leigir hjá öðrum án þinglýsts leigusamnings',
+    livesWithParents: 'Býr hjá foreldrum',
+    unknown: 'Annað',
+  }
```

Comment:
this belongs in formatters in shared, and than import with @island.is/financial-aid/shared/lib